packages/pulumi-aws/src/apps/api/ApiFileManager.ts

```diff
@@ -0,0 +1,174 @@
+import path from "path";
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+// @ts-ignore
+import { getLayerArn } from "@webiny/aws-layers";
+import { PulumiApp } from "@webiny/pulumi-sdk";
+
+import { Vpc } from "./ApiVpc";
+import { createLambdaRole } from "./ApiLambdaUtils";
+
+interface FileManagerParams {
+    fileManagerBucketId: pulumi.Input<string>;
+    awsAccountId: pulumi.Input<string>;
```

Comment:
I see `awsAccountId` is being defined here but not used anywhere. Is this OK? Can we double check?